test/test_gdalutilities.jl

```diff
@@ -286,4 +233,4 @@ end
         @test AG.height(ds_warped) == 100
         @test AG.shortname(AG.getdriver(ds_warped)) == "MEM"
     end
-end
+end
```

Comment:
Might be good to include a newline at the end of the file in the future -- but I think it'll get cleaned up by JuliaFormatter, so it doesn't matter too much.